lib/modules/showImages.js

```diff
@@ -1773,27 +1784,67 @@ function videoAdvanced(options) {
 		}
 	});
 
-	element.collapse = () => {
+	let playWhenVisibleInterval;
+
+	function playWhenVisible() {
+		const visibleThreshold = 0.33;
```

Comment:
I think we should play videos that have any percentage on the screen -- doing so would cost a marginal amount of performance (at most two videos, one at the top and bottom, assuming no overlap) but would marginally improve responsiveness, and I think that wins here.